### ISSUE TITLE
Small text on PatternFly.org?

### ISSUE THREAD

**Issue by catmanjan** [patternfly/patternfly-3#31]
Not sure if PatternFly.org represents a default PatternFly theme, but the text is tiny, verging on non-compliance with WCAG. Consider using a more readable font or increasing the font size by default.

**Comment by catmanjan** [comment]
Also I see issue #3 set this as resolved, but 12px OpenSans is still tiny, should be at least 12pt.
![untitled](https://cloud.githubusercontent.com/assets/137350/3760200/0e671640-1872-11e4-96be-c83adefe4f62.png)

**Comment by rhamilto** [comment]
Thank you for the feedback, @catmanjan.  We discussed this as a team and are choosing to keep things as they are for the time being.
